src/parser/FStar.Parser.Dep.fs

```diff
@@ -126,27 +126,30 @@ type dep_node = {
 }
 type dependence_graph = //maps file names to the modules it depends on
      | Deps of smap<dep_node> //(dependences * color)>
+type parsing_data = list<dependence> * list<dependence> * bool
 type deps = {
-    dep_graph:dependence_graph;              //dependences of the entire project, not just those reachable from the command line
-    file_system_map:files_for_module_name;   //an abstraction of the file system
-    cmd_line_files:list<file_name>;          //all command-line files
-    all_files:list<file_name>;               //all files
-    interfaces_with_inlining:list<module_name> //interfaces that use `inline_for_extraction` require inlining
+    dep_graph:dependence_graph;                 //dependences of the entire project, not just those reachable from the command line
+    file_system_map:files_for_module_name;      //an abstraction of the file system
+    cmd_line_files:list<file_name>;             //all command-line files
+    all_files:list<file_name>;                  //all files
+    interfaces_with_inlining:list<module_name>; //interfaces that use `inline_for_extraction` require inlining
+    parse_results:smap<parsing_data>
```

Comment:
comment here, following the style of the previous fields

Reply:
Done, https://github.com/FStarLang/FStar/commit/f5ba9096b86f8a0cf4243c587cb2ef8f1ccbb140#diff-bb2dbf1990bd066ee7fcfad0a2fd8b3f.